ibm-eventstore/src/main/scala/akka/stream/alpakka/ibm/eventstore/scaladsl/EventStore.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.ibm.eventstore.scaladsl
+
+import scala.concurrent.Future
+
+import akka.Done
+import akka.NotUsed
+import akka.stream.scaladsl.Flow
+import akka.stream.scaladsl.Keep
+import akka.stream.scaladsl.Sink
+
+import com.ibm.event.oltp.EventContext
+import com.ibm.event.oltp.InsertResult
+import org.apache.spark.sql.Row
+
+object EventStoreSink {
+
+  /**
+   * Creates a sink for insertion of records into EventStore.
+   * @param databaseName Name of the database, database has to exist before the call to this function.
+   * @param tableName    Name of the table, database has to exist before the call to this function.
+   * @return             CompletionStage[Done]
```

Comment:
Remove, type is already in signature (and in addition to that, this is the wrong type)